lib/appsignal_phoenix/live_view.ex

```diff
@@ -18,21 +19,20 @@ defmodule Appsignal.Phoenix.LiveView do
           kind, reason ->
             stack = __STACKTRACE__
 
+            @tracer.set_params(MapFilter.filter(params))
+            @tracer.set_environment(Appsignal.Metadata.metadata(socket))
+
             _ =
               span
-              |> @span.set_sample_data("params", params)
-              |> @span.set_sample_data("environment", Appsignal.Metadata.metadata(socket))
               |> @span.add_error(kind, reason, stack)
```

Comment:
I see there's no `set_error` function on the Elixir tracer either. Like we did for the Node.js integration. We should add that in another PR. Let's create an issue for that.